### PR TITLE
1924166: improve man text of syspurpose --show

### PR DESCRIPTION
**Pull request by ptoscano**
Spell "JSON" properly, and simplify the wording about it.

### CODE REVIEW

File: man/subscription-manager.8 [hunk 1]
```diff
@@ -384,7 +384,7 @@ for the system.
 
 .TP
 .B --show
-Shows the system's current set of syspurpose preference. This is output in the form of a blob of json. Single-valued entries for which there is no value will be included in the output with a value of "". List entries which have no value will be included in the output with a value of "[]" (less the quotes).
+Shows the system's current set of syspurpose preference, formatted as JSON. Single-valued entries for which there is no value will be included in the output with a value of "". List entries which have no value will be included in the output with a value of "[]" (less the quotes).
```
Comment (jirihnidek) [inline man/subscription-manager.8:387]: English is not my first language, but shouldn't be the change just "formatted as JSON" (without comma at beginning)?

Comment (ptoscano) [reply man/subscription-manager.8:387]: Me neither :) I added the comma to ease a bit the reading of the sentence, otherwise it'd be a single non-trivial sentence without any pause (comma, semicolon, colon, etc).

Comment (jirihnidek) [reply man/subscription-manager.8:387]: @tlhmerry9 I would like to know opinion, because you are native speaker. :smiley:

Comment (tlhmerry9) [reply man/subscription-manager.8:387]: I think it reads better:
Shows the system's current set of syspurpose preferences formatted as JSON.

**Review by jirihnidek — APPROVED** [review]
ACK
